CONVENTIONS.MD

```diff
@@ -0,0 +1,68 @@
+# Git
+### Branch namen
+- lower case met een "-" ertussen\
```

Comment:
De \ staat nu ook in de tekst. Er staan er nog een paar in dit bestand, even weg halen